iroha_http_server/src/lib.rs

```diff
@@ -247,40 +318,20 @@ pub mod http {
                     match self.method.as_ref() {
                         GET_METHOD => {
                             if let Endpoint::Http(HttpEndpoint::Get(handler)) = endpoint {
-                                match handler
+                                let response = handler
                                     .call(state, path_params, query_params, self.clone())
-                                    .await
-                                {
-                                    Ok(response) => Some(response),
-                                    Err(err) => {
-                                        log::error!(
-                                            "Failed to handle get request {:?} with error: {}",
-                                            self,
-                                            err
-                                        );
-                                        Some(HttpResponse::internal_server_error())
-                                    }
-                                }
```

Comment:
I think we should still log if the handler fails. Maybe add a flag option to opt in/out of logging, but I think there should be logging for these cases.

Reply:
It can be added in `Into` conversion